client/my-sites/site-settings/form-general.jsx

```diff
@@ -397,44 +398,62 @@ module.exports = React.createClass( {
 
 		return (
 			<div>
-				<SectionHeader label={ this.translate( 'Site Profile' ) } />
+				<SectionHeader label={ this.translate( 'Site Profile' ) }>
+					<Button
+						compact={ true }
+						onClick={ this.submitForm }
+						primary={ true }
+						type="submit"
+						disabled={ this.state.fetchingSettings || this.state.submittingForm }>
```

Comment:
Just for the next time maybe you could close the Component in the same TAB where it was opened.
```jsx
    <Button
        compact={ true }
        onClick={ this.submitForm }
        primary={ true }
        type="submit"
        disabled={ this.state.fetchingSettings || this.state.submittingForm }
    >
        { this.state.submittingForm
            ? this.translate( 'Saving…' )
// ...
```